### ISSUE TITLE
UserWarning: indexing with dtype torch.uint8 is now deprecated

### ISSUE THREAD

**Issue by dongluyang** [zzzxxxttt/pytorch_simple_cornernet#10]
C:\Users\panpa\ai\pytorch_simple_CornerNet\utils\losses.py:61: UserWarning: indexing with dtype torch.uint8 is now deprecated, please use a dtype torch.bool instead. (Triggered internally at  C:\cb\pytorch_1000000000000\work\aten\src\ATen/native/IndexingUtils.h:28.)
  loss = sum([F.smooth_l1_loss(r[mask], gt_regs[mask], reduction='sum') / num for r in regs])
C:\Users\panpa\ai\pytorch_simple_CornerNet\utils\losses.py:45: UserWarning: indexing with dtype torch.uint8 is now deprecated, please use a dtype torch.bool instead. (Triggered internally at  C:\cb\pytorch_1000000000000\work\aten\src\ATen/native/IndexingUtils.h:28.)
  embd0 = embd0[mask].sum()
C:\Users\panpa\ai\pytorch_simple_CornerNet\utils\losses.py:47: UserWarning: indexing with dtype torch.uint8 is now deprecated, please use a dtype torch.bool instead. (Triggered internally at  C:\cb\pytorch_1000000000000\work\aten\src\ATen/native/IndexingUtils.h:28.)
  embd1 = embd1[mask].sum()
